### PR TITLE
[ISSUE #378]Translation for "数据集成-概念" Section in the v4.x & 5.0 Document (CN -> EN)

### LINKED ISSUE #378: Translation for "数据集成-概念" Section in the v4.x & 5.0 Document (CN -> EN)
The Chinese version of the "数据集成-概念" section in the v4.x & v5.0 document has now been finished. However, its English version is still incomplete.

We now require someone who knows both Chinese and English and understands the technical terms of Apache RocketMQ to build it. The translation work is not only copy/paste from translation tools but also requires you to have some basic knowledge of Apache RocketMQ and technical writing.

### PR DESCRIPTION
**Pull request by mxsm**
Please do not create a Pull Request without creating an issue first. 

## What is the purpose of the change

close #378 

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [ ] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

File: i18n/en/docusaurus-plugin-content-docs/current/08-数据集成/26RocketMQ Connect Concept.md [hunk 1]
```diff
@@ -1,28 +1,26 @@
...
 
 ## Task
 
-是Connector任务分片的最小分配单位，是实际将源数据源数据复制数据到RocketMQ（SourceTask），或者将数据从RocketMQ读取数据写入到目标系统（SinkTask）真正的执行者，Task是无状态的可以动态的启停任务，多个Task是可以并行执行的，Connector复制数据的并行度主要体现在Task数量上。
+Task is the minimum allocation unit of Connector task sharding, which is responsible for actually copying the source data to RocketMQ (SourceTask), or reading data from RocketMQ and writing it to the target system (SinkTask). Tasks are stateless, and can be started and stopped dynamically. Multiple tasks can be executed in parallel, and the parallelism of data copying by the Connector is mainly reflected in the number of tasks.
 
-![RocketMQ基本模型](../picture/32rocketmq-connect/Connector-Task-Concept.png)
+![RocketMQ基本模型](E:/develop/github/contribution/rocketmq-site/i18n/en/docusaurus-plugin-content-docs/version-5.0/picture/32rocketmq-connect/Connector-Task-Concept.png)
```
Comment (Oliverwqcwrw) [inline i18n/en/docusaurus-plugin-content-docs/current/08-数据集成/26RocketMQ Connect Concept.md]: The link seems is incorrect

File: i18n/en/docusaurus-plugin-content-docs/current/08-数据集成/26RocketMQ Connect Concept.md [hunk 2]
```diff
@@ -1,28 +1,26 @@
...
 
-通过Connect的Api也可以看到Connector和Task各自的职责，Connector实现时就已经确定数据复制的流向，Connector接收数据源相关的配置，taskClass获取需要创建的任务类型，通过taskConfigs指定最大任务数量，并且为task分配好配置。task拿到配置以后从数据源取数据写入到目标存储。
+Through Connect's API, you can also see the responsibilities of Connector and Task, Connector has determined the data copy flow when it is implemented, Connector receives data source related configuration, taskClass obtains the type of task to be created, and taskConfigs specifies the maximum number of tasks, and allocates configuration for tasks. After task gets the configuration, it reads data from the data source and writes it to the target storage.
 
-通过下面的两张图可以清楚的看到，Connecotr和Task处理基本流程。
+From the following two diagrams, it is clear to see the basic flow of processing for Connector and Task.
 
-![RocketMQ基本模型](../picture/32rocketmq-connect/Connector-Task-process.png)
+![RocketMQ基本模型](E:/develop/github/contribution/rocketmq-site/i18n/en/docusaurus-plugin-content-docs/version-5.0/picture/32rocketmq-connect/Connector-Task-process.png)
```
Comment (Oliverwqcwrw) [inline i18n/en/docusaurus-plugin-content-docs/current/08-数据集成/26RocketMQ Connect Concept.md]: The link seems is incorrect

File: i18n/en/docusaurus-plugin-content-docs/current/08-数据集成/26RocketMQ Connect Concept.md [hunk 3]
```diff
@@ -1,28 +1,26 @@
...
 
 
 ## Worker
 
-worker 进程是Connector和Task运行环境，它提供RESTFul能力，接受HTTP请求，将获取到的配置传递给Connector和Task。
-除此之外它还负责启动Connector和Task，保存Connector配置信息，保存Task同步数据的位点信息，负载均衡能力，Connect集群高可用，扩缩容，故障处理主要依赖Worker的负载均衡能力实现的。
+The worker process is the running environment for Connector and Task, it provides RESTful capabilities, accepts HTTP requests, and passes the obtained configuration to Connector and Task. In addition, it is responsible for starting Connector and Task, saving Connector configuration information, saving the position information of Task's synchronized data, and load balancing capability. High availability, scaling and fault handling of Connect clusters mainly rely on the load balancing capability of Worker.
 
-![RocketMQ基本模型](../picture/32rocketmq-connect/worker.png)
+![RocketMQ基本模型](E:/develop/github/contribution/rocketmq-site/i18n/en/docusaurus-plugin-content-docs/version-5.0/picture/32rocketmq-connect/worker.png)
```
Comment (Oliverwqcwrw) [inline i18n/en/docusaurus-plugin-content-docs/current/08-数据集成/26RocketMQ Connect Concept.md]: The link seems is incorrect

File: i18n/en/docusaurus-plugin-content-docs/version-5.0/08-数据集成/26RocketMQ Connect Concept.md [hunk 4]
```diff
@@ -1,28 +1,26 @@
...
-，或从RocketMQ读数据写入到目标系统,这种是SinkConnector。Connector决定需要创建任务的数量，从Worker接收配置传递给任务。
+The connector defines where the data is copied from and where it is copied to. It reads data from the source system and writes it to RocketMQ, which is the SourceConnector, or reads data from RocketMQ and writes it to the target system, which is the SinkConnector. The Connector decides the number of tasks to be created, and receives configuration from the Worker and passes it to the task.
 
 ## Task
 
-是Connector任务分片的最小分配单位，是实际将源数据源数据复制数据到RocketMQ（SourceTask），或者将数据从RocketMQ读取数据写入到目标系统（SinkTask）真正的执行者，Task是无状态的可以动态的启停任务，多个Task是可以并行执行的，Connector复制数据的并行度主要体现在Task数量上。
+Task is the minimum allocation unit of Connector task sharding, which is responsible for actually copying the source data to RocketMQ (SourceTask), or reading data from RocketMQ and writing it to the target system (SinkTask). Tasks are stateless, and can be started and stopped dynamically. Multiple tasks can be executed in parallel, and the parallelism of data copying by the Connector is mainly reflected in the number of tasks.
 
 ![RocketMQ基本模型](../picture/32rocketmq-connect/Connector-Task-Concept.png)
```
Comment (Oliverwqcwrw) [inline i18n/en/docusaurus-plugin-content-docs/version-5.0/08-数据集成/26RocketMQ Connect Concept.md]:
```suggestion
![RocketMQ Basic Model](../picture/32rocketmq-connect/Connector-Task-Concept.png)
```

File: i18n/en/docusaurus-plugin-content-docs/version-5.0/08-数据集成/26RocketMQ Connect Concept.md [hunk 5]
```diff
@@ -1,28 +1,26 @@
...
 
-通过Connect的Api也可以看到Connector和Task各自的职责，Connector实现时就已经确定数据复制的流向，Connector接收数据源相关的配置，taskClass获取需要创建的任务类型，通过taskConfigs指定最大任务数量，并且为task分配好配置。task拿到配置以后从数据源取数据写入到目标存储。
+Through Connect's API, you can also see the responsibilities of Connector and Task, Connector has determined the data copy flow when it is implemented, Connector receives data source related configuration, taskClass obtains the type of task to be created, and taskConfigs specifies the maximum number of tasks, and allocates configuration for tasks. After task gets the configuration, it reads data from the data source and writes it to the target storage.
 
-通过下面的两张图可以清楚的看到，Connecotr和Task处理基本流程。
+From the following two diagrams, it is clear to see the basic flow of processing for Connector and Task.
 
 ![RocketMQ基本模型](../picture/32rocketmq-connect/Connector-Task-process.png)
```
Comment (Oliverwqcwrw) [inline i18n/en/docusaurus-plugin-content-docs/version-5.0/08-数据集成/26RocketMQ Connect Concept.md]:
```suggestion
![RocketMQ Basic Model](../picture/32rocketmq-connect/Connector-Task-process.png)
```

File: i18n/en/docusaurus-plugin-content-docs/current/08-数据集成/26RocketMQ Connect Concept.md (hunk 2 above)
Comment (Oliverwqcwrw) [inline i18n/en/docusaurus-plugin-content-docs/current/08-数据集成/26RocketMQ Connect Concept.md]:
```suggestion
![RocketMQ Basic Model](E:/develop/github/contribution/rocketmq-site/i18n/en/docusaurus-plugin-content-docs/version-5.0/picture/32rocketmq-connect/Connector-Task-process.png)
```

File: i18n/en/docusaurus-plugin-content-docs/current/08-数据集成/26RocketMQ Connect Concept.md (hunk 1 above)
Comment (Oliverwqcwrw) [inline i18n/en/docusaurus-plugin-content-docs/current/08-数据集成/26RocketMQ Connect Concept.md]:
```suggestion
![RocketMQ Basic Model](E:/develop/github/contribution/rocketmq-site/i18n/en/docusaurus-plugin-content-docs/version-5.0/picture/32rocketmq-connect/Connector-Task-Concept.png)
```

**Review by Oliverwqcwrw — CHANGES_REQUESTED** [review]

File: i18n/en/docusaurus-plugin-content-docs/current/08-数据集成/26RocketMQ Connect Concept.md (hunk 1 above)
Comment (mxsm) [reply i18n/en/docusaurus-plugin-content-docs/current/08-数据集成/26RocketMQ Connect Concept.md]: my fault, i will fix it and re-submit later

**Review by Oliverwqcwrw — APPROVED** [review]

**Review by RongtongJin — APPROVED** [review]

**Pull request merged**